includes/blocks/class-sensei-global-blocks.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * File containing the class Sensei_Global_Blocks.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Sensei_Global_Blocks
+ */
+class Sensei_Global_Blocks extends Sensei_Blocks_Initializer {
```

Comment:
Do you think it would make sense to try to add a few tests for this class?

Reply:
I was thinking about adding some tests for it, but then I saw that we usually didn't add tests for these loader classes, like for single page, or single course etc, so didn't think about it farther. But it totally makes sense to add tests for this class. Do you think we add a card to add tests for this later and merge this now?

Reply:
Historically, I don't think we've done the best job of adding tests. 😄  We could add a new card for this and I'll go ahead and approve.